api/src/main/java/io/strimzi/api/kafka/model/KafkaUserStatus.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class KafkaUserStatus implements UnknownPropertyPreserving, Serializable {
+
+    private String state;
+
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("the state")
```

Comment:
Do we have any better description for this? Also, I didn't saw how we expect the User status to look like. Can you share more details?

Reply:
This file was included accidentally for an early experiment so it has been removed